README.markdown

```diff
@@ -11,6 +11,17 @@ Install
 Run `setup.py install` to install the module.
 
 
+Setup for development on Mac OS X
+---------------------------------
+
+For Mac OS X:
+1. Install `afl-fuzz` via homebrew: `brew install afl-fuzz`
+2. Edit the `Makefile` located in `fuzzing/`
+    - Replace `afl-gcc` with `afl-clang`
```

Comment:
:+1: Nice, forgot how ancient the GCC toolchain was on OS X. Could we just detect OS X in the `Makefile` and automatically use `afl-clang`?